documentation/pom.xml

```diff
@@ -14,5 +14,6 @@
 
   <modules>
     <module>jetty</module>
+    <module>jetty-documentation</module>
```

Comment:
There is also the `jetty-asciidoctor-extensions` module that's not activated and stuck at version `12.0.10-SNAPSHOT`. Shouldn't it be enabled too?